32_awk_lab/awklab.txt

```diff
@@ -1,12 +1,17 @@
 1. Write an awk one-liner that prints the file "shells" with a blank line after each line
 Answer: 
 
+awk '1;!(NR%1){print "";}' /etc/shells
```

Comment:
Works, but you can use a simplified version:

```
awk ' { print; print ""' } ' /etc/shells
```

Or alternatively with printf:

```
awk '{ printf("%s\n\n", $0) }' /etc/shells
```